paper/paper.md

```diff
@@ -38,9 +38,33 @@ TO DO
 
 TO DO
 
-# Mathematics
+# HyP3 Description
+
+## Development History
+HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2017 by a group of undergraduate University of Alaska Fairbanks students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows that ran on-premise and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
```

Comment:
I decided to add this part of https://github.com/ASFHyP3/hyp3-paper/pull/19 as a suggestion here, in case that PR gets rejected, as this is the only really important part.
```suggestion
HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2016 by a group of undergraduate students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows that ran on-premise and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
```